### PR TITLE
Jhalackova dasd deployment

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding the deployment procedure description of Micro on IBM Z.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

**Comment by lvicoun** [comment]
/home/jana/SUSE/DOCS/doc-modular/build/Micro-deployment-raw-images-ibmz/Micro-deployment-raw-images-ibmz_en.pdf

**Comment by lvicoun** [comment]
[Micro-deployment-raw-images-ibmz_en.pdf](https://github.com/SUSE/doc-modular/files/14775751/Micro-deployment-raw-images-ibmz_en.pdf)

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xmlns:trans="http://docbook.org/ns/transclusion">
+  <info>
+    <title>Deploying the raw disk image on the DASD disk</title><!-- can be changed via merge
+in the assembly -->
+    <!-- add author's e-mail -->
+    <meta name="maintainer" content="jsindelarova@suse@com" its:translate="no"/>
+    <abstract><!-- can be changed via merge in the assembly -->
+      <para>
+       The topic covers the procedure how t odeploy the image to the machine.
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:27]:
```suggestion
       The topic covers the procedure of deploying the image to the machine.
```

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      </para>
+    </abstract>
+  </info>
+  <para>
+    To deploy the &productname; to the 20GB DASD disk, proceed as follows:
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:32]:
```suggestion
    To deploy the &productname; to the 20&nbsp;GB DASD disk, proceed as follows:
```

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  </para>
+  <procedure>
+    <step>
+      <para>
+        Download the raw disk image using <command>wget</command> or <command>curl</command>. for example:
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:37]:
```suggestion
        Download the raw disk image using <command>wget</command> or
        <command>curl</command>. For example:
```

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    </step>
+    <step>
+      <para>
+        Extract the image:
+      </para>
+      <screen>&prompt.user;unpack xz -d <replaceable>BUILD_IDENTIFICATION</replaceable>.raw.xz</screen>
+    </step>
+    <step>
+      <para>
+        Export a variable that finds and stores the 20GB device name.
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:49]:
```suggestion
        Export a variable that finds and stores the 20&nbsp;GB device name.  
```

File: tasks/deployment-raw-procedure-ibmz.xml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    </step>
+    <step>
+      <para>
+        Shut down the running Linux:
+      </para>
+      <screen>&prompt.root;init 0</screen>
+    </step>
+    <step>
+      <para>
+        Start &productname; by booting the second DASD minidisc in x3270 terminal
```
Comment (dariavladykina) [inline tasks/deployment-raw-procedure-ibmz.xml:84]:
```suggestion
        Start &productname; by booting the second DASD minidisk in 
        an x3270 terminal:
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see my suggestions for this one. Thanks!

**Pull request merged**